.changes/unreleased/NOTES-20240410-161445.yaml

```diff
@@ -0,0 +1,6 @@
+kind: NOTES
```

Comment:
If we want to make it a breaking change, we are protected based on the wording in the README and the versioning not being `1.x`

https://github.com/hashicorp/terraform-plugin-docs?tab=readme-ov-file#disclaimer

Reply:
I'd vote we call it explicitly a breaking change, especially since we've already set that expectation with the breaking changes in [`v0.17`](https://github.com/hashicorp/terraform-plugin-docs/blob/main/CHANGELOG.md#0170-january-17-2024) and `v0.15`